pandas/io/json/json.py

```diff
@@ -408,6 +415,9 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
                 {"index": "row 2", "col 1": "c", "col 2": "d"}]}'
     """
 
+    if orient == 'table' and dtype is not None:
```

Comment:
In the case previously where a user was explicitly passing `dtype=None` with the table orient this would now fail, right? I don't think we want that?

Reply:
@WillAyd raise condition is if dtype is NOT None.

Reply:
Right - `False` is not `None` so if someone was passing `dtype=False` previously it will now raise, which I'm not sure is desirable. Why not just `if orient == 'table' and dtype:`?

Reply:
@WillAyd, you are right. I have just fixed it to allow `orient='table'` and `dtype=False`.